src/Entities/Entity.cpp

```diff
@@ -457,10 +457,15 @@ bool cEntity::DoTakeDamage(TakeDamageInfo & a_TDI)
 					case mtSilverfish:
 					{
 						a_TDI.RawDamage += static_cast<int>(ceil(2.5 * BaneOfArthropodsLevel));
-						// TODO: Add slowness effect
+						// The duration of the effect is a random value between 1 and 1.5 seconds at level I,
+						// increasing the max duration by 0.5 seconds each level, up to 3.5s with Bane of Arthropods V.
+						// Ref: https://minecraft.gamepedia.com/Enchanting#Bane_of_Arthropods
+						double MaxDuration = std::min(1 + BaneOfArthropodsLevel * 0.5, 3.5);
```

Comment:
1 + 5 * 0.5 = 3.5 anyway, so do we really want the min() statement? Bane of Arthropods should only appear naturally up to level 5, I believe, so it won't affect normal gameplay, but a plugin author may want to give a sword a super high Bane of Arthropods level if they want longer slowness for whatever reason.